app/controllers/test/components/student/tracks_list_controller.rb

```diff
@@ -6,9 +6,11 @@ def show
   def tracks
     return head :internal_server_error if params[:state] == "Loading" || params[:state] == "Error"
 
+    tags = params[:tags].to_s.split(",")
```

Comment:
Is this a pattern we're planning on using in general?

If so, I'd like to standardise it so we have a helper in JS land that serializes to this, and a helper in Rails land that deserializes. It feels a little too arbitary to me to have this LOC in multiple places without it being  formalised as a pattern.

That's not blocking this being merged, but if you agree, can you open an issue to do that when you merge please.

We should also consider what character to use - there's probably a better choice than a `,`, which might actually be part of the string in some places. Maybe a unicode character that we have confidence wouldn't be used in a string?

Reply:
Instead of using a `,`, use the HTML standard and push `tags` as an array: 

```
tags[]=foo&tags[]=bar

params[:tags]
# => ['foo', 'bar']
```

Reply:
@SleeplessByte Do you know of a widly used React (or JS or whatever) helper for this, which we should use?

Reply:
Ahhh. Got it. 

If you want to push it to a URL: `URLSearchParams` (modern browsers, or [polyfill](https://github.com/ungap/url-search-params))
If you want to push it as form data: `FormData` (built-in).

Reply:
@kntsoriano Shall we go with this instead to be more HTML compliant?

Reply:
FWIW, comma separated works too, but yeah, then you have to deal with a comma never being a character inside a tag.

Reply:
Thanks! Applied the changes now.